src/components/collapsible-section/collapsible-section.scss

```diff
@@ -1,5 +1,12 @@
 @import '../../style/variables';
 
+/**
+ * @prop --limel-collapsible-section-header-closed-background-color: background color for header when closed
+ * @prop --limel-collapsible-section-header-open-background-color: background color for header when open
+ * @prop --limel-collapsible-section-body-background-color: background color for body
+ * @prop --limel-collapsible-section-body-padding: padding for body
```

Comment:
These are quite long and it would be nice if we could make them a bit shorter
We have been very inconsistent with the naming of our css variables :smile: The only other ones that have a prefix are the ones in `limel-grid` and they are only prefixed with `lime` _not_ `limel`

Would the following be too generic?
```
--closed-header-background-color
--open-header-background-color
--body-background-color
--body-padding
```

Reply:
Oh, yeah… we don't actually need to prefix them, because we keep them local to the component element anyway (or possibly the parent of it, but same-same). I forgot about that. Yeah, your suggestion is much nicer!